### DISCUSSION TITLE
Tailwind with the docs theme

### DISCUSSION THREAD

**Discussion by Rednegniw** [shuding/nextra#1323] [Q&A]
Hi there! Is there a way to use Tailwind with Nextra? For example for my custom components in mdx and so on

**Comment by fuma-nama** [comment]
Yes you are able to configure tailwind css and use it for your website
Just the same way as normal next.js applications, remember to add mdx files to the `content` property in tailwind.config.js

**Reply by yaffalhakim1** [reply]
do i need to create my own globals.css? to put the tailwind base and another

**Reply by fuma-nama** [reply]
> do i need to create my own globals.css? to put the tailwind base and another

Yes, the css styles of nextra has been build into normal css files so you have to create your own one and import it in `_app.mdx` or `_app.tsx`

**Comment by dimaMachina** [comment]
https://nextra.site/docs/guide/advanced/tailwind-css
